### PR TITLE
Split superuser_login_as into djangoapp

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
Failure is `gitlog`; ignoring...

File: openedx/stanford/lms/urls.py [hunk 1]
```diff
@@ -19,8 +20,7 @@
 if settings.FEATURES.get('ENABLE_SUPERUSER_LOGIN_AS'):
     urlpatterns += [
         url(
-            r'^su_login_as/(?P<username>[\w.@+-]+)/?$',
-            'student.views.superuser_login_as',
-            name='impersonate',
+            r'',
+            include('openedx.stanford.djangoapps.superuser_login_as.urls'),
```
Comment (caesar2164) [inline openedx/stanford/lms/urls.py]: shouldn't the url include be the first thing in the `url()` function?

Comment (stvstnfrd) [reply openedx/stanford/lms/urls.py]: It's an `include`; it doesn't have a url itself.

Reference `lms/urls.py`

Comment (caesar2164) [reply openedx/stanford/lms/urls.py]: oh ok, didn't know it worked like that.

**Pull request merged**